keras_cv/layers/preprocessing3d/global_random_flipping_y.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing3d import base_augmentation_layer_3d
+from keras_cv.ops.point_cloud import wrap_angle_radians
+
+POINT_CLOUDS = base_augmentation_layer_3d.POINT_CLOUDS
+BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES
+
+
+class GlobalRandomFlippingY(base_augmentation_layer_3d.BaseAugmentationLayer3D):
```

Comment:
just a discussion here: since we use "horizontal" and "vertical" in keras layers and Tensorflow in general, would it be better to be consistent with that?
https://keras.io/api/layers/preprocessing_layers/image_preprocessing/random_flip/
https://www.tensorflow.org/api_docs/python/tf/image/random_flip_left_right

I'm not sure if that image concept applies to point concept though

Reply:
The point coordinators are defined based on (X, Y, Z) instead of (row, col, Z). Unlike 2D images, 3D points are sparse and unordered. I think it is better to use 'Y' instead of 'horizontal/vertical'.

Reply:
sounds good